api/v1beta2/cryostat_types.go

```diff
@@ -325,10 +325,10 @@ type ReportsServiceConfig struct {
 // DatabaseServiceConfig provides customization for the service handling
 // traffic for the cryostat application's database.
 type DatabaseServiceConfig struct {
-	// HTTP port number for the cryostat application's database.
+	// Port number for the cryostat application's database.
 	// Defaults to 5432.
 	// +optional
-	HTTPPort      *int32 `json:"httpPort,omitempty"`
+	Port          *int32 `json:"httpPort,omitempty"`
```

Comment:
Should we rename the JSON property here too? Do you envision ever needing another port in the service? Would JDBCPort be too specific, maybe DatabasePort?

Reply:
Ah thanks, I overlooked that JSON property. Renaming it to be more specific is a good future-proofing idea.